### PR TITLE
Added mappings for Dimension

### PR DESCRIPTION
**Pull request by Siphalor**
+ Some related rendering stuff

### CODE REVIEW

File: mappings/net/minecraft/server/world/ServerWorld.mapping [hunk 1]
```diff
@@ -11,13 +12,18 @@ CLASS ur net/minecraft/server/world/ServerWorld
...
 	METHOD B getDimensionalPersistentStateManager ()Lcjb;
+	METHOD D getForcedChunkIds ()Lit/unimi/dsi/fastutil/longs/LongSet;
 	METHOD a init (Lbcv;)V
 		ARG 1 levelInfo
 	METHOD a save (Lzc;ZZ)V
 		ARG 1 progressListener
 	METHOD ak shouldSpawnNpcs ()Z
 	METHOD al shouldSpawnAnimals ()Z
 	METHOD am sendBlockActions ()V
+	METHOD b setForcedChunk (IIZ)Z
```
Comment (asiekierka) [inline mappings/net/minecraft/server/world/ServerWorld.mapping]: setChunkForced

File: mappings/net/minecraft/world/dimension/Dimension.mapping [hunk 2]
```diff
@@ -1,20 +1,45 @@
 CLASS bth net/minecraft/world/dimension/Dimension
+	FIELD a moonPhaseToSize [F
```
Comment (asiekierka) [inline mappings/net/minecraft/world/dimension/Dimension.mapping]: Is this a static final field, or not static?

File: mappings/net/minecraft/world/dimension/Dimension.mapping [hunk 3]
```diff
@@ -1,20 +1,45 @@
...
+	METHOD a getSpawningBlockInChunk (Lbcd;Z)Let;
+		ARG 1 chunkPos
+		ARG 2 checkMobSpawnValidity
+	METHOD a onEntityAdded (Lut;)V
+	METHOD b getFogColor (FF)Lcmw;
+		ARG 1 skyAngle
+		ARG 2 tickDelta
+	METHOD b onEntityRemoved (Lut;)V
 	METHOD d getForcedSpawnPoint ()Let;
+	METHOD e getHorizontShadingRatio ()D
```
Comment (asiekierka) [inline mappings/net/minecraft/world/dimension/Dimension.mapping]: getHorizonShadingRatio?

File: mappings/net/minecraft/server/world/ServerWorld.mapping [hunk 4]
```diff
@@ -11,13 +12,18 @@ CLASS ur net/minecraft/server/world/ServerWorld
 	METHOD <init> (Lnet/minecraft/server/MinecraftServer;Ljava/util/concurrent/Executor;Lcjd;Lcjc;Lbti;Laff;Lvc;)V
 		ARG 1 server
 	METHOD B getDimensionalPersistentStateManager ()Lcjb;
+	METHOD D getForcedChunkIds ()Lit/unimi/dsi/fastutil/longs/LongSet;
```
Comment (kashike) [inline mappings/net/minecraft/server/world/ServerWorld.mapping]: `getForcedChunks`

File: mappings/net/minecraft/world/dimension/Dimension.mapping (hunk 2 above)
Comment (kashike) [reply mappings/net/minecraft/world/dimension/Dimension.mapping]: It's `static final`.

File: mappings/net/minecraft/server/world/ServerWorld.mapping (hunk 4 above)
Comment (Siphalor) [reply mappings/net/minecraft/server/world/ServerWorld.mapping]: Sure?
It actually returns only the indices of the corresponding array and not the real chunks

Comment (Siphalor) [reply mappings/net/minecraft/server/world/ServerWorld.mapping]: Updated it

Comment (kashike) [reply mappings/net/minecraft/server/world/ServerWorld.mapping]: For the record, it doesn't return "indices", it returns the hashes of the chunk xz.

File: mappings/net/minecraft/world/dimension/OverworldDimension.mapping [hunk 5]
```diff
@@ -2,3 +2,6 @@ CLASS btk net/minecraft/world/dimension/OverworldDimension
 	METHOD <init> (Lbcq;Lbti;)V
 		ARG 1 world
 		ARG 2 type
+	METHOD b (FF)Lcmw;
+		ARG 1 arg1
```
Comment (Gegy) [inline mappings/net/minecraft/world/dimension/OverworldDimension.mapping]: These names don't seem particularly useful

Comment (Siphalor) [reply mappings/net/minecraft/world/dimension/OverworldDimension.mapping]: Enigma does some strange things; I can't even see this method.

Comment (Gegy) [reply mappings/net/minecraft/world/dimension/OverworldDimension.mapping]: Oh, huh. Gotta look into that.

**Review by Gegy — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**